lib/include/jxl/encode.h

```diff
@@ -410,14 +408,14 @@ JxlEncoderFrameSettingsSetInfo(JxlEncoderFrameSettings* frame_settings,
  * Calling @ref JxlEncoderFrameSettingsSetInfo clears any name that was
  * previously set.
  *
- * @param frame_options set of options and metadata for this frame. Also
+ * @param frame_settings set of options and metadata for this frame. Also
  * includes reference to the encoder object.
  * @param frame_name name of the next frame to be encoded, as a UTF-8 encoded C
  * string (zero terminated).
  * @return JXL_ENC_SUCCESS on success, JXL_ENC_ERROR on error
  */
 JXL_EXPORT JxlEncoderStatus JxlEncoderFrameSettingsSetName(
```

Comment:
This function's documentation should state something about how long that string should live for ;)

Reply:
I suggest we immediately copy it so it is never an issue, but it would indeed be good to document that so you don't have to worry about it.

Reply:
Done